src/buildCommon.js

```diff
@@ -413,14 +413,26 @@ const spacingFunctions = {
         size: "0.277778em",
         className: "thickspace",
     },
+    "\\thickspace": {
+        size: "0.277778em",
+        className: "thickspace",
+    },
```

Comment:
Are these separate commands or would it make sense to use `defineMacro` to define `\thickspace` in terms of `\;`?

Reply:
Using `defineMacro` would work very well. I'll change it.